components/net/fetch/methods.rs

```diff
@@ -753,7 +881,7 @@ fn http_network_or_cache_fetch(request: Rc<Request>,
 
     // Step 18
     if response.is_none() {
-        response = Some(http_network_fetch(request.clone(), http_request.clone(), credentials_flag));
+        response = Some(http_network_fetch(http_request.clone(), http_request.clone(), credentials_flag, done_chan));
```

Comment:
I just noticed that this change is silly; http_network_fetch has a preexisting bug where it accepts two `Rc<Request>` arguments and ignores one of them; we should remove the ignored one now :)